ci/jenkins/test.sh

```diff
@@ -453,6 +441,18 @@ function deliver_antrea_windows {
                 ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "docker load -i antrea-windows.tar.gz"
             fi
         fi
+        # Some tests need us.gcr.io/k8s-artifacts-prod/e2e-test-images/agnhost:2.13 image but it is not for windows/amd64 10.0.17763
```

Comment:
The commit message didn't mention this, and I wonder how defering this step a while can help avoid the test failure.

Reply:
This change is to avoid the situation where the test images is randomly deleted during the image building process due to disk pressure. I explained this in point 1, also updated the commit message in this PR.

Reply:
Please keep PR description in sync with commit message. It doesn't mention this.
And I remember image cleanup is not triggered by docker build, but kubelet periodically. When the disk usage is more than 85%, kubelet will do image GC, search imageGCHighThresholdPercent in https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/.

So even deferring image pulling, it could still be cleaned up if kubelet happens to do a GC before using the image.

I think we should guarantee enough free disk to fix it steadly, instead of relying on when kubelet runs GC.

Reply:
Description updated. The Containerd testbed has been allocated sufficient disk space, while the Docker testbed will maintain its current setup as it will be deprecated in 1.13.